### DISCUSSION TITLE
lastThroughputIn & lastThroughputOut doesn't reset

### DISCUSSION THREAD

**Discussion by hardcpp** [OvenMediaLabs/OvenMediaEngine#1370] [Help]
Hello, i dont known if it's a bug but on the rest route http://{SRV}:{PORT}/v1/stats/current/vhosts/{VHOST} the fields lastThroughputIn & 
lastThroughputOut doesn't reset to 0 when no streams are running.

**Comment by getroot** [comment]
Since it is "last", it seems logical that it will be maintained until the vhost is deleted. The information in stream (vhosts/{Vhost}/apps/{app}/streams/{stream} would have been deleted, wouldn't it?

**Comment by hardcpp** [comment]
Then maybe could we have a "current" fields? or a global bandwidth stat on the whole server instead of just the vhost?

**Reply by getroot** [reply]
I'll add a task to our backlog to add the current field for all stats.
